experiment/contractNipopow.sol

```diff
@@ -26,7 +26,8 @@ contract Crosschain {
     mapping(bytes32 => bool) finalizedEvents;
 
     // Security parameters.
-    uint256 constant m = 15;
+    // Make tests easy, we'll come back on these two
+    uint256 constant m = 6;
```

Comment:
perhaps a good chance to initialise these in the constructor? At least m that we are changing here.

Reply:
Done